hydradx/tests/test_omnipool_amm.py

```diff
@@ -678,20 +686,27 @@ def test_swap_lrna(initial_state: oamm.OmnipoolState):
     for asset in feeless_state.asset_list:
         feeless_state.last_lrna_fee[asset] = 0
         feeless_state.last_fee[asset] = 0
-    feeless_swap_state, feeless_swap_agent = oamm.simulate_swap_lrna(feeless_state, old_agent, delta_ra, 0, i)
-    if oamm.asset_invariant(feeless_swap_state, i) != pytest.approx(oamm.asset_invariant(old_state, i)):
-        raise
 
     # Test with trader selling LRNA
-    new_state, new_agent = oamm.simulate_swap_lrna(old_state, old_agent, 0, delta_qa, i)
-    feeless_swap_state, feeless_swap_agent = oamm.simulate_swap_lrna(feeless_state, old_agent, 0, delta_qa, i)
```

Comment:
Why is this test changing?

Reply:
simulate_swap_lrna is now simulate_swap(tkn_buy/sell = 'LRNA'). Having everything go through swap() lets us be consistent with oracle updates, which was an issue for the notebook in this PR.